main.py

```diff
@@ -365,30 +443,35 @@ def _get_gs_data():
     return jsonify(result=final_gsheet)
   
 def getCsvSettings(csv_data: dict, format: str) -> dict:
-  csv_settings = []
-  csv_columns, csv_date_column = get_csv_columns(csv_data)
-  if not csv_date_column:
-    csv_settings.append('error')
-    csv_settings.append(
-        "'date' column not found. Add 'date' heading and use date format"
-        " 'yyyy-mm-dd' in the values"
-    )
-
-  else:
-    start_date, end_date = csv_get_date_range(csv_data, csv_date_column)
-    if _validate_date(start_date) and _validate_date(end_date):
-      csv_settings.append('ok')
-      csv_settings.append(csv_date_column)
-      csv_settings.append(csv_columns)
-      csv_settings.append(start_date)
-      csv_settings.append(end_date)
-      csv_settings.append(format)
-    else:
+  try:
```

Comment:
can you narrow the scope of this try block so it doesn't cover as much code

Reply:
Fixed